paymentsheet/src/main/java/com/stripe/android/link/theme/Color.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("unused")
```

Comment:
I'll revert this once the palette is finalized, planning on keeping the colors used in the palette and removing the rest.